collector/webui/webserver.go

```diff
@@ -27,14 +27,15 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
+var Registry = prometheus.NewRegistry()
```

Comment:
Could the registry be created and returned by `StartWebServer()` to `main()`, thus registry would become a local variable ?